requirements.txt

```diff
@@ -5,15 +5,15 @@ bson==0.5.10 ; python_version >= "3.8" and python_version < "4.0"
 certifi==2022.12.7 ; python_version >= "3.8" and python_version < "4.0"
 colorama==0.4.6 ; python_version >= "3.8" and python_version < "4.0" and sys_platform == "win32"
 h11==0.14.0 ; python_version >= "3.8" and python_version < "4.0"
-httpcore==0.16.2 ; python_version >= "3.8" and python_version < "4.0"
-httpx==0.23.1 ; python_version >= "3.8" and python_version < "4.0"
+httpcore==0.16.3 ; python_version >= "3.8" and python_version < "4.0"
+httpx==0.23.3 ; python_version >= "3.8" and python_version < "4.0"
 idna==3.4 ; python_version >= "3.8" and python_version < "4.0"
 loguru==0.5.3 ; python_version >= "3.8" and python_version < "4.0"
-numpy==1.23.5 ; python_version < "4.0" and python_version >= "3.8"
-pandas==1.5.2 ; python_version >= "3.8" and python_version < "4.0"
+numpy==1.24.2 ; python_version < "4.0" and python_version >= "3.8"
+pandas==1.5.3 ; python_version >= "3.8" and python_version < "4.0"
 pyarrow==8.0.0 ; python_version >= "3.8" and python_version < "4.0"
 python-dateutil==2.8.2 ; python_version >= "3.8" and python_version < "4.0"
-pytz==2022.6 ; python_version >= "3.8" and python_version < "4.0"
+pytz==2023.3 ; python_version >= "3.8" and python_version < "4.0"
 rfc3986[idna2008]==1.5.0 ; python_version >= "3.8" and python_version < "4.0"
```

Comment:
was this file autoupdated or manually updated?

Reply:
This was automatically updated by running `make install` which contains one step to update the libraries `poetry lock -n && poetry export --without-hashes > requirements.txt`